### PR TITLE
Label > Add demos and code examples

### PR DESCRIPTION
**Pull request by Blackbaud-SteveBrush**

### CODE REVIEW

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-indicators?_sp.spa=pr-label-demos](https://developer.blackbaud.com/skyux-indicators?_sp.spa=pr-label-demos)

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-indicators?_sp.spa=pr-label-demos](https://developer.blackbaud.com/skyux-indicators?_sp.spa=pr-label-demos)

File: src/app/public/modules/label/label.component.ts [hunk 1]
```diff
@@ -11,17 +15,18 @@ import {
 export class SkyLabelComponent {
 
   /**
-   * The type of label to display. Possible values are `info`, `success`, `warning` and `danger`.
+   * The type of label to display.
    * @required
    */
   @Input()
-  public set labelType(value: string) {
+  public set labelType(value: SkyLabelType) {
```
Comment (Blackbaud-AlexKingman) [inline src/app/public/modules/label/label.component.ts:22]: I see what you did there

**Review by Blackbaud-AlexKingman — APPROVED** [review]

File: src/app/public/modules/label/label.component.ts (hunk 1 above)
Comment (Blackbaud-SteveBrush) [reply src/app/public/modules/label/label.component.ts:22]: HA!

**Pull request merged**